### PR TITLE
added jwt token expiry and test

### LINKED ISSUE #144: Handle token expiration with JWTs
Right now we create JWTs that are permanent. We should either implement some store for sessions to make sure, that a token hasn't been expired or destroyed or we should add expiration (`exp`) to the token and check it when validating tokens. We need to handle it in subjects and management APIs. We should not do it for processors for now, but there should also be some more elaborate protection there later on (follow-up issue?).

### PR DESCRIPTION
**Pull request by Jakedavies**
closes #144

### CODE REVIEW

**Comment by Jakedavies** [comment]
Tests appear to be failing because travis doesn't have the new env. Need to check with maks on how the test env is setup.

File: cg/test/subjects/subjects.api.spec.js [hunk 1]
```diff
@@ -91,9 +91,27 @@ describe('Tests of subject giving consent', () => {
     );
   });
 
+  it('Should return an error if the jwt token is expired', async done => {
+    const token = await subjectJWT.sign({ test: '10' }, { expiresIn: 1 });
```
Comment (sindelio) [inline cg/test/subjects/subjects.api.spec.js]: test: '10' means ?
expiresIn: 1 is one second or millisecond?

File: cg/test/subjects/subjects.api.spec.js [hunk 2]
```diff
@@ -91,9 +91,27 @@ describe('Tests of subject giving consent', () => {
...
+
+    expect.assertions(2);
+
+    setTimeout(async () => {
```
Comment (sindelio) [inline cg/test/subjects/subjects.api.spec.js]: waiting 1,1 seconds right

Comment (Jakedavies) [reply cg/test/subjects/subjects.api.spec.js]: yeah timeout is 1100 ms

File: cg/test/subjects/subjects.api.spec.js (hunk 1 above)
Comment (Jakedavies) [reply cg/test/subjects/subjects.api.spec.js]: test:10 means nothing, just a payload 
expiresIn1 is 1 second yes

**Pull request merged**